app/controllers/sitemap_controller.rb

```diff
@@ -0,0 +1,107 @@
+class SitemapController < ActionController::Metal
+
+  include AbstractController::Rendering
+  include ActionController::MimeResponds
+  include ActionController::DataStreaming
+  include ActionController::RackDelegation
+  include ActionController::Rescue
+  include ActionController::Head
+  include ActionController::Redirecting
+
```

Comment:
Sorry, I checked my facts. Instrumentation alone is not enough. append_into_to_payload method also needs to be overridden. The version used in LandingPageController is:

```ruby
  # Override basic instrumentation and provide additional info for
  # lograge to consume. These are pulled and logged in environment
  # configs.
  def append_info_to_payload(payload)
    super
    payload[:host] = request.host
    payload[:community_id] = community(request)&.id
    payload[:current_user_id] = nil
    payload[:request_uuid] = request.uuid
  end
```